### ISSUE TITLE
Warning scheduler get_lr

### ISSUE THREAD

**Issue by olix86** [neuropoly/ivado-medical-imaging#214]
I get the following warning while training on spine generic : warnings.warn("To get the last learning rate computed by the scheduler, ""please use `get_last_lr()`.")

Look like we need to replace `get_lr` with `get_last_lr`
https://github.com/pytorch/pytorch/blob/4ed790d74223ff3af95bbf8585cb67664805b6b9/torch/optim/lr_scheduler.py#L231

**Comment by olix86** [comment]
Fixed by #213
